llvm/test/CodeGen/RISCV/tlsdesc-symbol.ll

```diff
@@ -0,0 +1,22 @@
+; RUN: llc -mtriple=riscv64 -relocation-model=pic -enable-tlsdesc < %s \
+; RUN:     | llvm-mc -triple=riscv64 -filetype=obj -o %t.o --position-independent \
```

Comment:
If `-o %t.o`, the pipe should not be used

Reply:
I don't think a new file is needed. You can additionally add `llvm-readelf -s` lines to tls-models.ll

Reply:
I used a separate file since the other checks were autogenerated.

Reply:
I think the latest version now avoids the output file. Thanks for catching that.

Reply:
Just to confirm, we're OK w/ the extra test file? or shall I add manual tests to `tls-model.ll`?